objects/src/transaction/proven_tx.rs

```diff
@@ -25,7 +25,7 @@ pub struct ProvenTransaction {
     /// A list of nullifiers for all notes consumed by the transaction.
     input_notes: InputNotes<Nullifier>,
 
-    /// The id and  metadata of all notes created by the transaction.
+    /// All notes created by the transaction.
```

Comment:
Let's update this to: "Notes created by the transaction. For private notes, this will contain only note headers, while for public notes this will also contain full note details."